docs/data/data-grid/state/state.md

```diff
@@ -66,40 +66,36 @@ Some selectors are yet to be documented.
 
 ## Save and restore the state
 
-> ⚠️ This feature isn't implemented yet. It's coming.
+The current state of the grid can be exported using `apiRef.current.exportState()`.
+It can then be restored by either passing it to the `initialState` prop or to the `apiRef.current.restoreState()` method.
+
+Watch out for controlled models and their callbacks (`onFilterModelChange` if you use `filterModel` for instance), the grid will call those callbacks when restoring the state.
+But if the callback is not defined or if calling it does not update the prop value, then the restored value will not be applied.
+
+### Restore the state with `initialState`
+
+> ⚠️ If you restore the page using `initialState` before the data is fetched, the grid will automatically move to the 1st page.
```

Comment:
We miss a small paragraph introducing what the following demo does.

Reply:
I added a small introduction